### DISCUSSION TITLE
I have difficulties enabling letsencrypt automatic certificates

### DISCUSSION THREAD

**Discussion by he-ftv** [globaleaks/globaleaks-whistleblowing-software#4316] [Q&A]
Hello,
I had to renew my HTTPS certificate. So far I have done it manually, but I would like to do it in the future with automatic configuration. Unfortunately I always get an error, see screenshot. The public IP address and DNS records are configured correctly and I checked our logs on the firewall and there is nothing blocked. Maybe there is still something to configure on Linux? Can you guys help me out? Thanks!

![image](https://github.com/user-attachments/assets/fbc6930f-9186-4fda-b847-bc5fd613bee9)

**Comment by evilaliv3** [comment]
@he-ftv: it seems that your firewall is currently blocking access on port 80.

The lesencrypt protocol is officially designed to work on port 80.

**Comment by he-ftv** [comment]
Hi, I don't even see traffic on port 80. All I see blocked are some weird random URL's (see screenshot). What exactly needs to be allowed for "Automatic configuration" ? Which URL's / Ports?

![image](https://github.com/user-attachments/assets/8acad2ec-6731-4e04-ab99-b6cb5d9f43ad)

**Reply by he-ftv** [reply]
I have now deactivated the URL filter on the firewall, but still get the error. Can you help me to identify the problem and fix it? Thanks a lot! @evilaliv3
